project.clj

```diff
@@ -23,7 +23,9 @@
                                     :password :env/clojars_password
                                     :sign-releases false}]]
 
-  :jvm-opts ["-Dclojure.main.report=stderr"]
+  :jvm-opts ~(cond-> ["-Dclojure.main.report=stderr"]
+               (not jdk8?) (conj "--add-opens"
+                                 "jdk.compiler/com.sun.tools.javac.tree=ALL-UNNAMED"))
```

Comment:
We need this flag in order to the doc comment parser to be available.

enrich-classpath will add it, soon.